deployment/bare_metal/.env.dist

```diff
@@ -68,7 +68,7 @@ FEDERATION_COMMUNITY_API_PORT=5000
 
 FEDERATION_CONFIG_VERSION=v1.2023-01-09
 # comma separated list of api-versions, which cause starting several federation modules
-FEDERATION_COMMUNITY_APIS=1_0,1_1,2_0
+FEDERATION_COMMUNITY_APIS=1_0,1_1
```

Comment:
it doesn't make sense to keep only the version 1_1 next to 1_0 even version 2_0 has real differences but not only a different version-nr.

Reply:
Don't we have the version number test covered through the ``1_0`` and we tests the minor version upgrade with the ``1_1``? And we had in the proof that ``2_0`` would work.
For me this ``1_0,1_1`` solution would be good!

Reply:
A single version-number for its own will not ensure the treatement of several different versions.
Now to remove the version 2_0, which really differes from 1_0 and 1_1 (the version 1_1 has no differenc against the version 1_0, it simply shows a solution how to avoid duplicate code if there are no changes against a previous version) seems not like an intelligent solution, because it is impossible to test against real different apis and not only different version-numbers.